filter-api/src/main/java/org/hypertrace/agent/filter/FilterRegistry.java

```diff
@@ -70,6 +78,27 @@ private static Filter load() {
     return new MultiFilter(filters);
   }
 
+  private static ClassLoader loadJars() {
+    List<StringValue> jarPaths = HypertraceConfig.get().getJavaagent().getFilterJarPathsList();
+    List<File> files = new ArrayList<>();
```

Comment:
Do we need to create file objects? It does not seem to be needed. We can create directly URLs